bot/common/tasks/tasks.py

```diff
@@ -1,25 +1,56 @@
-import threading
 import logging
 import asyncio
+import discord
+import pytz
 
 from abc import ABC, abstractmethod
 from datetime import timedelta, datetime, time
+from discord.ext import tasks, commands
 
 from bot.common.tasks.weekly_contributions import send_weekly_contribution_reports
-from bot.common.cache import Cache, RedisCache
+from bot.common.cache import Cache
+from bot.constants import BotTasks as TaskConstants
 
 logger = logging.getLogger(__name__)
 
-TASK_LOOP_INTERVAL_MINUTES = 10
-REPORT_LAST_SENT_DATETIME_CACHE_KEY = "contribution_report_last_sent"
+# TODO: tz?
```

Comment:
Good, question would this mess up what is currently in the cache. Would this add any benefit

Reply:
I think for now it would be easier to just disregard the tz, and manually set the time via .env to EDT/EST. But, maybe other guilds would want the reporting time configurable.

Reply:
that is, if no tz is specified on the cache entry, we just assume the tz is the same as the system ( if we add it later)